README.md

```diff
@@ -30,3 +30,7 @@ script:
 addons:
   chrome: stable # make sure you have Chrome available
 ```
+
+## Related Projects
+
+- [Lighthouse CI GitHub Action](https://github.com/treosh/lighthouse-ci-action) - Audit URLs using Lighthouse and monitor performance with Lighthouse CI.
```

Comment:
```suggestion
- [Lighthouse CI GitHub Action](https://github.com/treosh/lighthouse-ci-action) - Automatically run Lighthouse CI on every PR with GitHub Actions, no infrastructure required.
```

Reply:
I have a feeling that all related projects here will be using Lighthouse CI to audit urls so tried to tailor it a bit for the github actions part :)